wiki/Accuracy/en.md

```diff
@@ -1,132 +1,38 @@
 
-For more details, see [Score][Score wikilink].
+![Accuracy formula for osu!standard](img/accuracy_standard.png)
 
-![osu! Ranking Panel](img/standard.jpg "osu! Ranking Panel")
-![osu!Taiko Ranking Panel](img/taiko.jpg "osu!Taiko Ranking Panel")
-![osu!CtB Ranking Panel](img/catch.jpg "osu!CtB Ranking Panel")
-![osu!Mania Ranking Panel](img/mania.jpg "osu!Mania Ranking Panel")
+Reference for one hit circle:
 
-### Performance Graph
+```
+300 -> 300 / 300 = 1   = 100.00%
+100 -> 100 / 300 = 1/3 =  33.00%
+50  ->  50 / 300 = 1/6 =  16.67%
+0   ->   0 / 300 = 0   =   0.00%
+```
 
-A graph of the performance over the course of a play.
+## osu!taiko
 
-Additional information is shown when hovering the in-game cursor over it:
+In osu!taiko, accuracy is calculated by taking the sum of the note accuracy divided by the number of notes. The note accuracy are as follows: a GREAT (良) counts as 100%, GOOD (可) as 50% (half), and MISS/BAD (不可) as 0% (which also breaks the combo). Drum rolls and spinners do not influence the accuracy. In other words, this can be summarized with the following formula:
 
-![Performance Graph](img/tr.jpg "Performance Graph")
+![Accuracy formula for osu!taiko](img/accuracy_taiko.png)
 
-#### Accuracy
+## osu!catch
 
-Term | Meaning
----- | -------
-Error | **These two values represent how far off the early hits are on average, and how far off the late hits are on average**. The higher the [OD value][OD wikilink] of the beatmap played, the lower these will have to be to do well.
-Unstable Rate | **The value represents how consistently the timing of the hits**, with lower numbers being better (top players often score below 100). Note that the value measures **consistency**, and not _accuracy_, so consistently in hitting 15ms early is the same as consistently in hitting on time. The formula is essentially the standard deviation of the hit errors (in milliseconds) multiplied by 10.
+In osu!catch, accuracy is calculated by taking the total of non-spinner hit objects collected divided by the total number of non-spinner objects. All hit objects have the same value, except for bananas, as they are part of the spinner object. In other words, this can be summarized with the following formula:
 
-#### Spin
+![Accuracy formula for osu!catch](img/accuracy_catch.png)
 
-_Spin is only for [osu!standard][osu! wikilink]_
+*Note for API users: to calculate the accuracy in osu!catch, number of droplets are under `count50` and number of missed droplets are under `countkatu`.*
 
-Term | Meaning
----- | -------
-Speed | **Average spinning speed on all the spinner(s) in the beatmap**. Max is the highest rpm(rounds per minute) achieved in one of the beatmap's spinner(s).
-Unstable Rate | **Values based on average against Max deviation**, the lower the better. Formula is unknown.
+## osu!mania
 
-**Notes:-**
+In osu!mania, accuracy is calculated similarly to [osu!standard](#osu!standard). In other words, this can be summarized with the following formula:
```

Comment:
```
In osu!mania, accuracy is calculated similarly to [osu!standard](#osu!standard).

In other words, this can be summarized with the following formula:
```